### DISCUSSION TITLE
Path error after running conda activate moose

### DISCUSSION THREAD

**Discussion by linearcoffeecup** [idaholab/moose#28697] [Q&A Getting Started]
### Check these boxes if you have followed the posting rules.

- [X] I have consulted the [Posting Guidelines](https://github.com/idaholab/moose/discussions/18270).
- [X] I have searched the [Discussions Forum](https://github.com/idaholab/moose/discussions) and [MOOSE Framework Troubleshooting](https://mooseframework.inl.gov/help/troubleshooting.html) and have not found what I was looking for
- [X] Q&A Getting Started is the most appropriate category for my question (trouble installing, beginner user, ...)

## Error Message after typing conda activate moose

xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

## Other

This error message also shows up after running cd ~/projects/moose/test
./run_tests -j 6.

This test also finds: ModuleNotFoundError: No module named 'hit'

## Computer
MacOS Sonoma silicone

**Comment by GiudGiud** [comment]
hello

looks like you need to install command line tools from xcode
https://www.freecodecamp.org/news/install-xcode-command-line-tools/

**Reply by linearcoffeecup** [reply]
Thank you.  I installed the command line tools and the path error went away.

I ran the tests again.  There were failures for anything related to web_server_control (everything else passed or skipped).  Is this Ok or do these web_server_control failures need to be fixed?

**Reply by GiudGiud** [reply]
no need to fix them for now

**Reply by linearcoffeecup** [reply]
Than you for all your help.
